### PR TITLE
makes civ bounties not completely broken

### LINKED ISSUE #55989: bounty dont work: cant select a bounty @arcanemusic
153683

pressing on bounty to select does nothing except a noise

@ArcaneMusic

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
(i told you to testmerge it dammit)

## Why It's Good For The Game
Fixes #55989

## Changelog
:cl:
fix: civilian bounty terminals have had a hotfix 
/:cl:

### CODE REVIEW

File: code/game/machinery/civilian_bounties.dm [hunk 1]
```diff
@@ -130,7 +130,7 @@
 	pot_acc.bounties = crumbs
 
 /obj/machinery/computer/piratepad_control/civilian/proc/pick_bounty(choice)
-	if(inserted_scan_id?.registered_account)
+	if(!inserted_scan_id || !inserted_scan_id.registered_account)
```
Comment (Jared-Fogle) [inline code/game/machinery/civilian_bounties.dm:133]: I don't understand, why didn't you just invert the existing null coalescence check?

Comment (spessbro) [reply code/game/machinery/civilian_bounties.dm:133]: changed it and the machine worked again

Comment (stylemistake) [reply code/game/machinery/civilian_bounties.dm:133]:
```suggestion
	if(!inserted_scan_id?.registered_account)
```

**Comment by ArcaneMusic** [comment]
Can you check that the return value for this fix will return with the correct bounty result?

**Comment by spessbro** [comment]
> Can you check that the return value for this fix in the UI will return with the correct bounty result?

done and tested

**Pull request merged**